OracleGoldenGate/21c/install-prerequisites.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+## Copyright (c) 2021, Oracle and/or its affiliates.
+set -e
+
+##
+##  i n s t a l l - p r e r e q u i s i t e s . s h
+##  Installation of prerequisite software
+##
+
+packages=(findutils java-1.8.0-openjdk jq libaio libnsl nginx procps-ng python3 python3-requests python3-psutil tar unzip)
+manager=$(command -v microdnf || command -v dnf || command -v yum)
+
+function success() {
+    echo "Packages installed after ${sequence} attempts"
+    rm -rf /var/cache/yum /var/log/yum* /tmp/yum*
+    useradd ogg
+    exit 0
+}
+
+for sequence in $(seq 3); do
+    "${manager}" -y install "${packages[@]}" && success "${sequence}"
```

Comment:
You should be using modules for OL8 which is annoying with `microdnf` and why we don't recommend it. After rebasing to `oraclelinux:8`, please use `dnf module install nginx` and `dnf module install python36` for those components.